src/ex1/index.html

```diff
@@ -1,12 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <title>Exercise 1</title>
+    <title>Todo App</title>
     <link rel="stylesheet" href="style.css" />
 </head>
 <body>
-
-<h1>Todo list</h1>
+<div class="todoAppContainer">
+    <h1 class="titleH1">Todo list </h1>
+    <div class="taskInputAndTaskAddContainer">
+        <input type="text" id="todoTaskInput" placeholder="Add your new todo">
+        <button type="submit" class="addTaskButton" id="addTaskButtonId" onclick=onAddTaskButton()>+</button>
+    </div>
+    <div class="todoTasksListContainer">
+        <ul class="taskList">
+        </ul>
+    </div>
+    <div class="pendingAndClearButtonContainer">
```

Comment:
what if you'd like to add more functionalities to this container, would you change the name again? :)
better call it something like "bottom-menu-container" and then inside decide on the components